.github/workflows/upgrade-one-python-dependency.yml

```diff
@@ -0,0 +1,75 @@
+name: Upgrade one Python dependency
+
+on:
+  workflow_dispatch:
+     inputs:
+       branch:
+         description: 'Target branch to create requirements PR against'
+         required: true
+         default: 'master'
+         type: string
+       package:
+         description: 'Name of package to upgrade'
+         required: true
+         type: string
+
+defaults:
+  run:
+    shell: bash
```

Comment:
Is this necessary to specify bash shell explicitly?

Reply:
Apparently if you specify it, you get stricter error checking! https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

I thought I had added a comment to that effect. (I'll add one.)